easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1.2-fosscuda-2020b.eb

```diff
@@ -0,0 +1,181 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.1.2'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+builddependencies = [
+    # CMake/Doxygen/SWIG are required for building OpenMM
+    ('CMake', '3.18.4', ),
+    ('Doxygen', '1.8.20', ),
+    # required for installing dm-tree
+    ('Bazel', '3.7.2', ),
+    ('SWIG', '4.0.2', ),
+]
+
+dependencies = [
+    ('Python', '3.8.6', ),
+    ('SciPy-bundle', '2020.11'),
+    ('PyYAML', '5.3.1', ),
+    ('TensorFlow', '2.4.1'),
```

Comment:
AlphaFold [requires TensorFlow 2.5.0](https://github.com/deepmind/alphafold/blob/57a2455e5fab752d8df430ae1d0334e1a21b548c/requirements.txt#L13), since we already have TensorFlow 2.5.0 in 2020b I suggest to upgrade to it to follow upstream more closely.

Reply:
@lexming thanks, I was not aware of that. But how do I deal with "AssertionError: ['TensorFlow'] is not false : No multi-variant deps found" in the checks?

Reply:
you need to add a test suite exception to test/easyconfigs/easyconfigs.py (around line 488)